app/components/post_draft/uploads/index.tsx

```diff
@@ -109,21 +109,21 @@ function Uploads({
         } else {
             errorHeight.value = ERROR_HEIGHT_MIN;
         }
-    }, [uploadFileError]);
+    }, [errorHeight, uploadFileError]);
 
     useEffect(() => {
         if (files.length) {
             containerHeight.value = CONTAINER_HEIGHT_MAX;
             return;
         }
         containerHeight.value = CONTAINER_HEIGHT_MIN;
-    }, [files.length > 0]);
+    }, [containerHeight, files.length]);
```

Comment:
Nitpick:

If we create a new variable `hasFiles`, we might not run this effect so often, and may be slightly more intention revealing. (Not that this is neither hard to understand or has any performance issue though)

```
const hasFiles = Boolean(files.length);

useEffect(() => {
    if (hasFiles) {...}
    ...
}, [containerHeight, hasFiles]);
```